command/agent/config/test-fixtures/config.hcl

```diff
@@ -0,0 +1,32 @@
+pid_file = "./pidfile"
+
+auto_auth {
+	method {
```

Comment:
Could the format for the config be like:
```
method "aws-iam" {
    ...
}

sink "file" {
    ...
}
```

Reply:
I had it that way but changed it. It makes the config parsing more complicated but also there wasn't a situation where other values didn't need to be specified anyways. But, I could change it back.

Reply:
I like that this way is more similar to the existing vault config/policy format, and it feels more HCL-y if that's a thing.

Reply:
Fixed! You can now do either.